### PR TITLE
refactor: update orbit components to 0.7.0

### LINKED ISSUE #659: Update Orbit components to 0.7.0
FYI: please note this requires update of orbit components also on frontend when integrating these changes

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
closes #659<br/><br/><br/><url>LiveURL: https://smartfaq-update-orbit-components.surge.sh</url>

### CODE REVIEW

**Comment by vborszeki** [comment]
the UI for the legs is broken

screen shot 2018-08-17 at 17 50 30

File: src/SingleBookingPage/bookingItem/AccordionLegTypeIcon.js [hunk 1]
```diff
@@ -26,15 +26,15 @@ const LegTypeIcon = (props: IconProps) => {
   let icon;
   switch (type) {
     case bookingLegTypes.AIRCRAFT:
-      icon = <AirplaneDown customColor="#adb9c5" height="12" />;
+      icon = <AirplaneDown customColor="#adb9c5" size="small" />;
```
Comment (vborszeki) [inline src/SingleBookingPage/bookingItem/AccordionLegTypeIcon.js]: please change the size prop to medium

File: src/SingleBookingPage/bookingItem/AccordionLegTypeIcon.js [hunk 2]
```diff
@@ -26,15 +26,15 @@ const LegTypeIcon = (props: IconProps) => {
...
       break;
 
     case bookingLegTypes.TRAIN:
-      icon = <Train customColor="#00a991" height="12" />;
+      icon = <Train customColor="#00a991" size="small" />;
```
Comment (vborszeki) [inline src/SingleBookingPage/bookingItem/AccordionLegTypeIcon.js]: the same here

File: src/SingleBookingPage/bookingItem/AccordionLegTypeIcon.js [hunk 3]
```diff
@@ -26,15 +26,15 @@ const LegTypeIcon = (props: IconProps) => {
...
       break;
 
     case bookingLegTypes.BUS:
-      icon = <Bus customColor="#00a991" height="12" />;
+      icon = <Bus customColor="#00a991" size="small" />;
```
Comment (vborszeki) [inline src/SingleBookingPage/bookingItem/AccordionLegTypeIcon.js]: and here too 🙂

File: src/SingleBookingPage/bookingItem/AccordionBodyLeg.js [hunk 4]
```diff
@@ -71,20 +71,20 @@ const AccordionBodyLeg = (props: LegProps) => {
...
-          <Calendar customColor="#2f363c" height="20" />
+          <Calendar customColor="#2f363c" size="medium" />
         </div>
         <div className="date">
           {FormatDate({ dateString: initialDeparture })}
         </div>
         <AccordionLegCities leg={props.leg} />
         <div className="layover">
-          <Clock customColor="#7c8b99" height="16" />
+          <Clock customColor="#7c8b99" size="small" />
```
Comment (vborszeki) [inline src/SingleBookingPage/bookingItem/AccordionBodyLeg.js]: this size prop should be also medium, now it's breaking the alignment of the text

screen shot 2018-08-17 at 18 16 58

**Comment by jaroslav-kubicek** [comment]
all issues should be resolved @vborszeki

File: src/SingleBookingPage/bookingItem/AccordionLegCitiesInfo.js [hunk 5]
```diff
@@ -38,14 +38,15 @@ const citiesInfoStyle = css`
 
 type Props = {| leg: AccordionLegCitiesInfo_leg |};
 
+const renderIcon = Icon => <Icon size="small" color="secondary" />;
```
Comment (oreqizer) [inline src/SingleBookingPage/bookingItem/AccordionLegCitiesInfo.js]: this is lazy, plz don't

File: src/StaticFAQ/breadcrumbs/__test__/CustomBreadcrumbs.test.js [hunk 6]
```diff
@@ -2,10 +2,10 @@
 
 import * as React from 'react';
 import { mount } from 'enzyme';
-import ThemeProvider from '@kiwicom/orbit-components/lib/Theming/ThemeProvider';
 
 import WithRouter from '../../../common/__tests__/helpers/ComponentWithRouter.ignore';
 import CustomBreadcrumbs from '../CustomBreadcrumbs';
+import ThemeProvider from '../../../common/ThemeProvider';
```
Comment (oreqizer) [inline src/StaticFAQ/breadcrumbs/__test__/CustomBreadcrumbs.test.js]: you don't need `ThemeProvider` in tests anymore anywhere

Comment (jaroslav-kubicek) [reply src/StaticFAQ/breadcrumbs/__test__/CustomBreadcrumbs.test.js]: fixed, thanks!

**Comment by jaroslav-kubicek** [comment]
FYI: please note this requires update of orbit components also on frontend when integrating these changes

**Pull request merged**